examples/ec2-instance-feature-demo/blueprint.yaml

```diff
@@ -266,6 +266,9 @@ node_templates:
               MinCount: 1
               ImageId: { get_attribute: [ cloudify_manager_ami, aws_resource_id ] }
               InstanceType: m3.large
+            modify_instance_attribute_args:
+              SourceDestCheck:
+                Value: false
```

Comment:
Why do we need to add this to configure operations ?

Reply:
Most of the time when someone wants this feature however, they need it during install. `modify_instance_attribute` requires that the instance is not "Started" state. So, it needs to happen either directly after instantiation or later (during the process described before.)

Reply:
Thanks